sites/all/modules/hr/hr_search/hr_search.install

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @file
+ * Update file for the Search feature.
+ */
+
+/**
+ * Implements hook_update_N().
+ * Re enables field_infographic_type field under search api fields.
+ */
+
+function hr_search_update_7001() {
+  $index = search_api_index_load('default_node_index', TRUE);
+
+  $fields = array(
+    'options' => $index->options,
+  );
+
+  $fields['options']['fields']['field_infographic_type'] = array(
+    'type' => 'integer',
+    'entity_type' => 'taxonomy_term',
+  );
+  search_api_index_edit($index->id, $fields);
```

Comment:
I guess here we're doing whatever the manual operation would do, right?
Could we do this for the rest of the fields that make the feature appear as overriden as well? Maybe in a different ticket.